db/migrate/20230404075703_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string, default: "editor"
```

Comment:
The active record migration docs aren't clear on this. Does the `default: "editor"` mean that all subsequent inserts of users to the `users` table will be given `editor` role unless otherwise provided when they're created? Does it mean that all existing user entries will be set to `editor`? If so, is that safe, do we have something to remember to change this since it might not be the right starting role. Perhaps a default of 'none' might be safer, then we actively update them to grant them a role?

Reply:
@danworth 

> Does the default: "editor" mean that all subsequent inserts of users to the users table will be given editor role unless > otherwise provided when they're created?
Correct

> Does it mean that all existing user entries will be set to editor?  If so, is that safe, do we have something to 
> remember to change this since it might not be the right starting role
Correct, for the time being it should be safe as we are simply allowing signed in users to do what they currently being doing. We haven't really discussed a users with no role, TBH this is kind of where "trial" accounts are more likely to come into play. When we get to that point we can do another migration to change the default role going forward.